yt/frontends/stream/tests/test_stream_species.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+from yt.loaders import load_uniform_grid
+from yt.testing import assert_allclose_units
+
+
+def test_stream_species():
+    arr = np.random.uniform(size=(32, 32, 32))
+
+    data = {
+        "density": (arr, "g/cm**3"),
+        "H_p0_fraction": (0.37 * np.ones(arr.shape), "dimensionless"),
+        "H_p1_fraction": (0.37 * np.ones(arr.shape), "dimensionless"),
+        "He_fraction": (0.24 * np.ones(arr.shape), "dimensionless"),
+        "CO_fraction": (0.02 * np.ones(arr.shape), "dimensionless"),
```

Comment:
just curious, where do these magic numbers (0.37, 0.24, 0.02) come from ? not that it impacts the test in any way.

Reply:
I'm assuming here that 0.74 is the total H mass fraction, 0.24 is the total He mass fraction, and the rest is CO (the latter of which I am pretty sure is true nowhere in the universe but they add up to 100% and I wanted to throw in a molecule for good measure). Then I just split the H mass fraction evenly between fully ionized and neutral hydrogen because reasons.